agiverse/agent/memory/embedding.py

```diff
@@ -0,0 +1,29 @@
+from typing import List, Optional
+import numpy as np
+import litellm
+from dotenv import load_dotenv
+import logging
+from tenacity import retry, stop_after_attempt, wait_exponential
+
+logger = logging.getLogger(__name__)
+
+load_dotenv()
```

Comment:
it's better not to use dotenv in the package because it might cause side effect. It should be user's responsibility to set environment variables.